circle.yml

```diff
@@ -10,12 +10,8 @@ dependencies:
   pre:
     - google-chrome --version
     - curl -L -o google-chrome.deb https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
-    - for v in $(curl http://archive.ubuntu.com/ubuntu/pool/main/n/nss/ | grep "href=" | grep "libnss3.*deb\"" -o | grep -o "libnss3.*deb" | grep "3.28" | grep "14.04"); do curl -L -o $v http://archive.ubuntu.com/ubuntu/pool/main/n/nss/$v; done && rm libnss3-tools*_i386.deb libnss3-dev*_i386.deb
     - sudo dpkg -i google-chrome.deb || true
-    - sudo dpkg -i libnss3*.deb || true
     - sudo apt-get update
-    - sudo apt-get install -f || true
-    - sudo dpkg -i libnss3*.deb
```

Comment:
@VictorBjelkholm this is breaking chrome on CI

Reply:
@diasdavid yeah, because js-ipfs is still using old Ubuntu version. Once this PR is ready to be merged (all CIs except CircleCI passing), we'll upgrade the version and confirm all CIs again and then merge this.

Reply:
I'm not sure if you need me to do something or if it is all part of your plan. Let me know if you need my help :)

Reply:
@VictorBjelkholm shall I go ahead and merge this PR?